src/main/java/com/linecorp/armeria/server/http/tomcat/TomcatServiceBuilder.java

```diff
@@ -238,6 +268,19 @@ private static String validateHostname(String hostname) {
     }
 
     /**
+     * Sets a {@link Consumer} that performs additional configuration operations against
+     * the Tomcat {@link StandardServer} created by a {@link TomcatService}. This method can be invoked
+     * multiple times to add multiple {@code configurator}s.
+     */
+    public TomcatServiceBuilder configurator(Consumer<? super StandardServer> configurator) {
```

Comment:
Here's the list of the properties I will check. Let me know if you think there should be more:

```java
        // 1. Check if the default connector has not been removed
        // 2. Check if the default service has not been removed
        // 3. Check if a new service has not been added
        // 4. Check if engine has not been changed
        // 5. Check if baseDir has not been changed
        // 6. Check if Server.port has not been changed
        // 7. Check if the default host and context have not been removed
```